source/assets/stylesheets/scss/_test-drive-gocd.scss

```diff
@@ -27,12 +27,13 @@ body.test-drive-gocd {
     }
   }
 
-  .content-header {
+  .content--header {
```

Comment:
@veerababukona any specific reason for using `--` instead of `-` ?

Reply:
I am following BEM naming conventions.
Reference: http://getbem.com/naming/